src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -395,16 +395,20 @@ class StreamGroup extends React.Component<Props, State> {
         {hasInbox && (
           <React.Fragment>
             {inboxTabActive && (
-              <Box width={80} mx={2} className="hidden-xs hidden-sm">
-                <InboxReason data={data} />
-              </Box>
+              <ReasonBox width={80} mx={2} className="hidden-xs hidden-sm">
+                <BadgeWrapper>
+                  <InboxReason data={data} />
+                </BadgeWrapper>
+              </ReasonBox>
             )}
-            <Box width={150} mx={2} className="hidden-xs hidden-sm">
-              <StyledTimes
-                lastSeen={data.lifetime?.lastSeen || data.lastSeen}
-                firstSeen={data.lifetime?.firstSeen || data.firstSeen}
-              />
-            </Box>
+            <TimesBox width={130} mx={2} className="hidden-xs hidden-sm">
```

Comment:
I think this needs to be larger because sometimes reason and times badges touch
![Screen Shot 2020-11-11 at 5 47 41 PM](https://user-images.githubusercontent.com/15015880/98884961-ffed0980-2445-11eb-80a8-fc97b312857d.png)

Reply:
Thanks for the feedback, really wanted to get them smaller